mitmproxy/log.py

```diff
@@ -1,7 +1,91 @@
-from dataclasses import dataclass
+import logging
+import os
+import warnings
+
+from mitmproxy.contrib import click as miniclick
+from mitmproxy.utils import human
+
+ALERT = logging.INFO + 1
+"""
+The ALERT logging level has the same urgency as info, but
+signals to interactive tools that the user's attention should be
+drawn to the output even if they're not currently looking at the
+event log.
+"""
+logging.addLevelName(ALERT, "ALERT")
```

Comment:
Thoughts on also adding a `logging.alert(...)` function to make it look and behave the same way as built-in levels?
https://stackoverflow.com/a/35804945

Reply:
I've been on that SO page as well, but intentionally decided against this as it does not really play well with mypy. We could add it for quick-and-dirty addons and then not use it ourselves, but not sure if that is particularly useful.

Reply:
Ok. My only fear is that nobody will use ALERT, because it requires this extra little step - compared to a possibly auto-completed function.